exercises/diffie-hellman/.meta/version

```diff
@@ -0,0 +1,2 @@
+1.0.0
```

Comment:
Not sure if we should have a `version` for this, seeing as there is no canonical data for this exercise yet (this is where we normally source version numbers from).

Reply:
True, exercises ported to Java track with no canonical data don't have ```version```.